sklearn/linear_model/tests/test_ridge.py

```diff
@@ -553,6 +553,30 @@ def test_ridge_gcv_sample_weights(
     assert_allclose(gcv_ridge.intercept_, kfold.intercept_, rtol=1e-3)
 
 
+def test_ridge_gcv_stored_predictions():
+    X, y = make_regression()
+    ridge_cv = RidgeCV(fit_intercept=False, scoring="neg_mean_squared_error",
+                       store_cv_values=True)
+    ridge_cv.fit(X, y)
+    ridge = Ridge(fit_intercept=False)
+    loo_pred = cross_val_predict(ridge, X, y, cv=len(X))
+    assert_allclose(loo_pred, ridge_cv.cv_values_[:, 1])
+
+
+def test_ridge_gcv_decision_function_scoring():
```

Comment:
What is this test testing?

Reply:
I think that this test was intended to check the equivalence between `score=None` and `score=scorer` where scorer compute the mean squared error